SilMock/Google/Service/Directory/Resource/Groups.php

```diff
@@ -15,6 +15,19 @@ public function __construct(?string $dbFile = null)
         parent::__construct($dbFile, 'directory', 'groups');
     }
 
+    public function get(string $groupKey): ?GoogleDirectory_Group
+    {
+        $mockGroupsObject = new Groups($this->dbFile);
+        $groupsObject = $mockGroupsObject->listGroups();
+        $groups = $groupsObject->getGroups();
```

Comment:
This seems a bit odd (convoluted), but probably done to match Google, so I'm just noting it, and not holding anything up due to it.

Reply:
Right, because Google does Google/...._Groups which listGroups() returns, and that isn't a Group array, which getGroups() returns.